cluster/gce/util.sh

```diff
@@ -589,6 +589,22 @@ function kube-up {
   find-release-tars
   upload-server-tars
 
+  if [[ ${KUBE_SHARE_MASTER:-} == "true" ]]; then
```

Comment:
I think that @fgrzadkowski may be OOO for a while so let's not hold this up on his PR.